packages/actor-http-cache/lib/headerConversionHelpers.ts

```diff
@@ -0,0 +1,66 @@
+import type {
+  Headers as HeaderHash,
+  Request as CacheRequest,
+  Response as CacheResponse,
+} from 'http-cache-semantics';
+
+/**
+ * Convert a request into the old version of the request that doesn't use the
+ * Headers object
+ * @param request the request
+ * @returns A request that has hash header
+ */
+export function requestToRequestWithHashHeaders(
+  request: Request,
+): CacheRequest {
+  return { ...request, headers: headersToHash(request.headers) };
+}
+
+/**
+ * Convert a response into the old version of the response that doesn't use
+ * the Headers object
+ * @param response the response
+ * @returns A response that has hash header
+ */
+export function responseToResponseWithHashHeaders(
+  response: Response,
+): CacheResponse {
+  return { ...response, headers: headersToHash(response.headers) };
+}
+
+/**
+ * Convert the given headers object into a raw hash.
+ * @param headers A headers object.
+ */
+export function headersToHash(headers: Headers): HeaderHash {
```

Comment:
I guess this one can be removed in favor of https://github.com/comunica/comunica/blob/master/packages/bus-http/lib/ActorHttp.ts#L56

Reply:
Yep, done